src/main/java/net/mcreator/element/types/Biome.java

```diff
@@ -93,6 +93,10 @@
 	public List<String> defaultFeatures;
 
 	public List<SpawnEntry> spawnEntries;
+	public List<SpawnEntry> spawnMonsters;

```

Comment:
Why did you add these new lists, spawn entries are already classified into these types in the original list?

Reply:
I added them because when I tried to list each entity, I always got `null` when I had two different spawn types. It was the only solution I found to resolve the problem.

Reply:
Please try to figure out why this was happening, adding four new lists holding the same information is not acceptable solution and I will not be able to merge this PR.

Reply:
I'm trying to find a solution since I made the first version of the file (so a few months ago) and even after a lot of tests, I still don't find why it happens (I have a small idea, but I don't know/think we can fix it with another solution).